docs/Readme.md

```diff
@@ -0,0 +1,5 @@
+# Docs
```

Comment:
Can you change the file name to `README`?

Just to be consistent with the first one.